Source/FullScreenPokemon.d.ts

```diff
@@ -1056,7 +1056,17 @@ declare module FullScreenPokemon {
         /**
          * Activates a Function to perform an HM move outside of battle.
          */
-        partyActivate?: (player: IPlayer, pokemon: IPokemon) => boolean;
+        partyActivate?: (player: IPlayer, pokemon: IPokemon) => void;
+
+        /**
+         * Activates a Function to make sure an HM move can be used outside of battle.
+         */
+        partyActivateCheckThing?: (player: IPlayer, pokemon: IPokemon, move: IMoveSchema) => void;
```

Comment:
Why add this? It's just a reference to FSP.partyActivateCheckThing in your usage.